Modules/SubModule_Population/xASL_adm_GzipAllFiles.m

```diff
@@ -1,7 +1,7 @@
-function xASL_adm_GzipAllFiles(ROOT, bFolder, bUseLinux)
+function xASL_adm_GzipAllFiles(x, bFolder, bUseLinux)
```

Comment:
Maybe I'm contradicting myself, but it would be good if this function can be used without the internal memory table of ExploreASL (x), so I would keep the first input argument as it was. E.g., I sometimes use this function after initializing ExploreASL but without loading a study.

Reply:
Sure, we just need to have a way to determine the path to this external executable. For this we need the ExploreASL x.MyPath. I can commit an alternative.

Reply:
Please checkout the changes in b507565 and the test below.

Reply:
Sorry, but I wouldn't do this. I would keep "Root" as first argument as it was, for those using this outside of ExploreASL. If we need the ExploreASL path (or x) as additional input, you can add this as fourth, new, input argument.

The point is that the folder in which you want to zip all files, could be the study root folder but could easily be a single subject, or the full ABBA folder, if you want to apply this to a lot of studies. So it is rather a tool where in 1 specific case (the population module) we use `x.D.ROOT`. So I would revert `x` to `ROOT`, and add `x` as last argument.